modules/ai/assets/js/editor/utils/editor-integration.js

```diff
@@ -68,13 +69,12 @@ export const renderLayoutApp = ( options = {
 
 	const { colorScheme, isRTL } = getUiConfig();
 
+	const bodyStyle = window.elementorFrontend.elements.$window[ 0 ].getComputedStyle( window.elementorFrontend.elements.$body[ 0 ] );
+
```

Comment:
why do you move it? I would keep it in the same place is possible, to have less diff